website_rentals/static/src/components/RentalWizard.xml

```diff
@@ -62,7 +62,7 @@
                                         subcomponents defined in this block. -->
                                     <div t-att-class="(state.startDateInput &amp;&amp; state.endDateInput &amp;&amp; !state.loading) ? '' : 'd-none'">
                                         <div t-if="!state.quantityAvailable" class="row">
-                                            <p class="text-danger pt-2">No quantity available.</p>
+                                            <p class="text-danger pt-2">No availability.</p>
```

Comment:
@holdenrehg update the `.po` files as this involves a term change

Reply:
👍🏻 https://github.com/Yenthe666/rental/commit/555cc083ce55c5738aad00228b34dd071c7dd132